public/locales/uk/support.json

```diff
@@ -1,3 +1,14 @@
 {
-    "Frequently Asked Questions": "Поширені питання"
+    "Frequently Asked Questions": "Поширені питання",
+    "See also": "Дивися також",
+    "Removing your account": "Видалення вашого акаунту",
+    "Please write in English if possible.": "Будь ласка, пишіть на  англійській мові, якщо це можливо.",
+    "Please ensure you are connected to internet and try again.": "Будь ласка, переконайтесь, що ви під’єднані до інтернету і спробуйте ще раз.",
+    "Name": "Ім'я",
+    "Message": "Повідомлення",
+    "I’m just a small website robot but I’ve sent your message to our support people. Expect them to get back to you very soon!": "Я лише маленький робот цього  вебсайту, але я відправив ваше повідомлення нашій команді підтримки. Очікуйте, вони швидко зв'яжуться із вами!",
+    "If you or someone you know have witnessed or been a victim of a crime, please report it to the police immediately.": "Якщо ви, чи хтось кого ви знаєте став свідком чи жертвою злочину, будь ласка, негайно повідомте у поліцію",
+    "Email": "Емейл",
+    "Contact us": "Контакти",
+    "<0>Become a volunteer</0> and make a difference!": "Ставай волонтером"
```

Comment:
Missing `<0>` — commented https://hosted.weblate.org/translate/trustroots/support/uk/?checksum=d001f82387c1969b#comments

Reply:
Fixed in Weblate.